homeassistant/components/freebox/const.py

```diff
@@ -48,3 +53,28 @@
     "vg_console": "mdi:gamepad-variant",
     "workstation": "mdi:desktop-tower-monitor",
 }
+
+ATTR_DETECTION = "detection"
+ATTR_ACTIVATION = "activation"
+ATTR_QUALITY = "quality"
+ATTR_SENSITIVITY = "sensitivity"
+ATTR_THRESHOLD = "threshold"
+ATTR_FLIP = "flip"
+ATTR_TIMESTAMP = "timestamp"
+ATTR_VOLUME = "volume"
+ATTR_SOUND_DETECTION = "sound_detection"
+ATTR_SOUND_TRIGGER = "sound_trigger"
+ATTR_RTSP = "rtsp"
+ATTR_DISK = "disk"
+
+MODEL_PIR = "F-HAPIR01A"
+MODEL_CAMERA = "F-HACAM01A"
+MODEL_DWS = "F-HADWS01A"
+MODEL_KBF = "HAKFB01A"
+MODEL_ALARM = "F-MSEC07A"
+MODEL_RTS = "RTS"
+MODEL_IOHOME = "IOHome"
```

Comment:
This comment https://github.com/home-assistant/core/pull/88104#discussion_r1112226679 does not refer to consts but  the assignation the the model attr

```suggestion
```